one_fm/api/tasks.py

```diff
@@ -1,5 +1,4 @@
-import itertools
-from datetime import timedelta
+import itertools,time

```

Comment:
@Talleyrand333 Is time being used anywhere?

Reply:
@ks093 thanks for pointing this out, It was used to test how long the process takes. it has been removed